config-templates/ibm/ibm-object-storage-plugin/2.2/custom-parameters.json

```diff
@@ -0,0 +1,42 @@
+[
+    {
+        "description": "Release name of the chart",
+        "displayname": "Helm Chart Release Name",
+        "name": "helm-release-name",
+        "category": "config",
+        "required": "true"
+    },
+    {
+        "description": "Helm Chart Additional Parameters (Optional)",
+        "displayname": "Helm Chart Additional Parameters (Optional)",
+        "name": "parameters",
+        "category": "config",
+        "required": "false",
+        "type": "csv",
+        "mutable": "true"
+    },
+    {
+        "description": "COS License (Apache 2.0)",
+        "displayname": "COS License (Apache 2.0)",
+        "name": "license",
+        "category": "config",
+        "required": "false",
+        "default": "true"
+    },
+    {
+        "description": "COS Endpoint (for more info refer - https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-endpoints)",
+        "displayname": "COS Endpoint",
+        "name": "cos-endpoint",
+        "category": "config",
+        "required": "false",
+        "default": "https://s3.us.cloud-object-storage.appdomain.cloud"
```

Comment:
Set `"default"` = "" and `"required"` = `"true"`

Reply:
only `"required": true` is sufficient. Remove `default` key